drivers/gpio/gpio-eic-sprd.c

```diff
@@ -653,7 +653,6 @@ static int sprd_eic_probe(struct platform_device *pdev)
 		return ret;
 	}
 
-	platform_set_drvdata(pdev, sprd_eic);
```

Comment:
your `Signed-off-by` tag is  with ` Coardos Andrei <traiandrei@yahoo.com>`
maybe change it to `Andrei Coardos  <traiandrei@yahoo.com>` ?

Reply:
same comment [as this one](https://github.com/commodo/linux/pull/14/commits/a3d1e02ae2d9d03a27c59fb2986822d62c57123a#r1263002893)

the driver was added in 2018;
and the `platform_set_drvdata()` call is just a copy+paste slip-up